src/oc_erchef/raml-docs/policies.yml

```diff
@@ -34,8 +30,8 @@ documentation:
       displayName: group - Policy Group Name
       type: string
   get:
-    description: |
-      Retrieve the Policy document
+    is: [ versioned ]
```

Comment:
Idk I thought literally all of our endpoints returned the version header? That's all that means. It returns the version header.